File: ReptiCare/Settings+CoreDataProperties.m
//
//  Settings+CoreDataProperties.m
//  ReptiCare
//
//  Created by Kevin De Koninck on 24/09/2016.
//  Copyright © 2016 Kevin De Koninck. All rights reserved.
//

#import "Settings+CoreDataProperties.h"

@implementation Settings (CoreDataProperties)

+ (NSFetchRequest<Settings *> *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"Settings"];
}

@dynamic disablePushNotifications;
@dynamic disableSounds;
@dynamic language;
@dynamic uniqueID;
@dynamic weightUnit_;

@end

File: ReptiCare/Settings+CoreDataProperties.swift
//
//  Settings+CoreDataProperties.swift
//  ReptiCare
//
//  Created by Kevin De Koninck on 24/09/2016.
//  Copyright © 2016 Kevin De Koninck. All rights reserved.
//

import Foundation
import CoreData


extension Settings {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<Settings> {
        return NSFetchRequest<Settings>(entityName: "Settings");
    }

    @NSManaged public var disablePushNotifications: NSNumber?
    @NSManaged public var disableSounds: NSNumber?
    @NSManaged public var language: String?
    @NSManaged public var uniqueID: NSNumber?
    @NSManaged public var weightUnit_: NSNumber?

}
